Jonathan branch

SomeGuyNamedJon opened the pull request:
Updating

e76971072 committed:
add more items in each room

SomeGuyNamedJon committed:
Wall collision fixed

Camera also zoomed out a touch

SomeGuyNamedJon committed:
Merge branch 'JonathanBranch' of https://github.com/e76971072/Final-Project-Game into JonathanBranch

e76971072 committed:
working now with collider

e76971072 committed:
Merge branch 'JonathanBranch' of https://github.com/e76971072/Final-Project-Game into JonathanBranch

SomeGuyNamedJon committed:
Merge branch 'master' into JonathanBranch

SomeGuyNamedJon committed:
Added Textures to rooms

Rooms are now textured, albeit a bit crudely. Files organized and more 2d assets imported

SomeGuyNamedJon committed:
Merge remote-tracking branch 'origin/KevinBranch' into JonathanBranch

SomeGuyNamedJon committed:
Fixed floating room error on this branch and gave closed room a texture

SomeGuyNamedJon committed:
Mild Beautification

Fixed layer masks and respective textures to make the game look a lot prettier. Changed the camera background color to make room transitions less jarring.

isbd commented:
Checked it out, looks good. When merging be sure to take the master's RBRoom.prefab in the conflict.

SomeGuyNamedJon committed:
Merge branch 'master' into JonathanBranch

SomeGuyNamedJon committed:
Inadvertantly broke bullet prefab (FIXED)

Changed the tag that the room trigger has, updated the code. Moved player sprite to z = -1 to allways have him and everything he spawns appear above the environment textures. Use the negative z direction from now on when you want textures to appear above other textures.

SomeGuyNamedJon committed:
Missed some tags to update; Deleted the "CameraControl" tag completely
